src/main/java/org/jitsi/jicofo/BridgeSelector.java

```diff
@@ -729,11 +829,70 @@ public int compareTo(BridgeState o)
             boolean otherOperational = o.isOperational();
 
             if (meOperational && !otherOperational)
+            {
                 return -1;
+            }
             else if (!meOperational && otherOperational)
+            {
                 return 1;
+            }
+
+            return this.getEstimatedVideoStreamCount()
+                - o.getEstimatedVideoStreamCount();
+        }
+
+        private int getEstimatedVideoStreamCount()
+        {
+            int videoStreamEstimate
+                = videoStreamCount != null ? videoStreamCount : 0;
+
+            if (videoStreamCountDiff != null)
```

Comment:
yes.. I guess I should store a snapshot in a var, before evaluating. Will that work ? Probably I should revise some other calculations around BridgeState as well